src/__test__/list.test.js

```diff
@@ -43,4 +43,43 @@ describe('add and remove', () => {
     todoList.deleteActivity(1);
```

Comment:
Please add instructions to run the project locally. Instructions should include running `npm install` and `npm start`. Please also add instructions to run the tests.

Reply:
> Done